src/annoylib.h

```diff
@@ -204,70 +224,6 @@ static inline void normalize(float *v, int f) {
     v++;
   }
 }
-
-template<typename T, typename Random, typename Distance, typename Node>
```

Comment:
actually realized the AVX version I just removed had some speedups that were unique to it - when it updates `iv` and `jv` it actually won't benefit from AVX in the current version. I'm tempted to make that sacrifice since tree building isn't usually a big issue (Annoy is already very fast) and I found the AVX version to be much harder to read (and creates a bit of code redundancy)